dockstore-common/src/test/java/io/dockstore/common/PipHelperTest.java

```diff
@@ -10,8 +10,8 @@
 public class PipHelperTest {
     @Test
     public void convertSemVerToAvailableVersion() {
-        Assert.assertEquals("1.7.0", PipHelper.convertSemVerToAvailableVersion(PipHelper.DEV_SEM_VER));
-        Assert.assertEquals("1.7.0", PipHelper.convertSemVerToAvailableVersion(null));
+        Assert.assertEquals("1.10.0", PipHelper.convertSemVerToAvailableVersion(PipHelper.DEV_SEM_VER));
+        Assert.assertEquals("1.10.0", PipHelper.convertSemVerToAvailableVersion(null));
```

Comment:
Ideally there would be tests for `PipHelper.convertSemVerToAvailableVersion("1.10.0")` and `PipHelper.convertSemVerToAvailableVersion("1.10.1)` if I understand the possible parameters that can be passed in.

Reply:
Well, it's a good thing you mentioned this. A test failed